File: sentimental-mario-less/mario.py
from cs50 import get_int


def main():

    # Get height

    size = get_height()

    # Print blocks

    print_blocks(size)


def get_height():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            return height


def print_blocks(size):

    # Initial if statement for a single block
    if size == 1:
        print("#")

    # Else if statement for additional layer (based on height input of 2)
    elif size == 2:
        print(" #\n##")

    # Else if statement for additional layer (based on height input of 3)
    elif size == 3:
        print("  #\n ##\n###")

    # Else if statement for additional layer (based on height input of 4)
    elif size == 4:
        print("   #\n  ##\n ###\n####")

    # Else if statement for additional layer (based on height input of 5)
    elif size == 5:
        print("    #\n   ##\n  ###\n ####\n#####")

    # Else if statement for additional layer (based on height input of 6)
    elif size == 6:
        print("     #\n    ##\n   ###\n  ####\n #####\n######")

    # Else if statement for additional layer (based on height input of 7)
    elif size == 7:
        print("      #\n     ##\n    ###\n   ####\n  #####\n ######\n#######")

    # Else if statement for additional layer (based on height input of 8)
    elif size == 8:
        print("       #\n      ##\n     ###\n    ####\n   #####\n  ######\n #######\n########")


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);
void print_blocks(int size);

int main(void)
{
    // Get height

    int h = get_height();

    // Print blocks

    print_blocks(h);
}

// Individual function for height prompt
int get_height(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    return h;
}

// Individual function for block print
void print_blocks(int size)
{

// Initial if statement for a single block

    if (size == 1)
    {
        printf("#");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 2)

    else if (size == 2)
    {
        printf(" #\n##");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 3)

    else if (size == 3)
    {
        printf("  #\n ##\n###");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 4)

    else if (size == 4)
    {
        printf("   #\n  ##\n ###\n####");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 5)

    else if (size == 5)
    {
        printf("    #\n   ##\n  ###\n ####\n#####");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 6)

    else if (size == 6)
    {
        printf("     #\n    ##\n   ###\n  ####\n #####\n######");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 7)

    else if (size == 7)
    {
        printf("      #\n     ##\n    ###\n   ####\n  #####\n ######\n#######");
        printf("\n");
    }

// Else if statement for additional layer (based on height input of 8)

    else if (size == 8)
    {
        printf("       #\n      ##\n     ###\n    ####\n   #####\n  ######\n #######\n########");
        printf("\n");
    }
}